### ISSUE TITLE
[Usability] A11y_Manage.IIS.Net_Authorization_Keyboard: Keyboard focus should be inside the dropdown and it should not come out of the drop down when drop down is expanded.

### ISSUE THREAD

**Issue by Accessibility56** [Microsoft/IIS.WebManager#195]
GitHubTags: #A11yTCS; #A11yUsable; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #Keyboard; #C+E-H; #.NETWebsites; #Edge; #Chrome

**Environment Details:**
**URL:** https://manage.iis.net/webserver/authorization
**Browser Details:** Edge(42.17134.1.0),Chrome(68.0.3440.106)
Keyboard

**Repro Steps:**
1.Download and install "Microsoft IIS Administration" application.
2.Enable Internet information Services.
3.Open "Microsoft IIS" application using the URL: https://manage.iis.net/.
4.Tab till Web Server and press enter.
5.Tab till "Authorization"  and press enter.
6.Tab till "More" button and press enter.
7.Verify focus order inside drop down.

**Actual Result:**
Keyboard focus should be inside the dropdown and it should not come out of the drop down when drop down is expanded.

**Expected Result:**
Keyboard focus should be inside the drop down when it is expanded. Focus should come out of it only when we collapsed it. Other wise screen reader should read at least out of list item when it is reading contents out side of drop down.

**User Impact:** 
Screen Readers may think that whatever screen reader is reading is related to the drop down items only,  because they did not collapse the drop down and still drop down is in expanded state.

**Attachment:**
[Bug_195_Authorization_Focus is coming out of the pop up.pptx](https://github.com/Microsoft/IIS.WebManager/files/2335813/Bug_195_Authorization_Focus.is.coming.out.of.the.pop.up.pptx)

**Comment by jhkimnew** [comment]
Fixed
